packages/form-field-scripts/src/paths.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+import path from 'path'
+import fs from 'fs'
+
+const appDirectory = fs.realpathSync(process.cwd())
+const resolveApp = (relativePath) => path.resolve(appDirectory, relativePath)
+
+const moduleFileExtensions = [
+    // "web.mjs",
+    // "mjs",
+    // "web.js",
+    'js',
+    // "web.ts",
+    // "ts",
+    // "web.tsx",
+    // "tsx",
+    'json',
+    // "web.jsx",
+    'jsx',
+]
+
+const resolveModule = (resolveFn, filePath) => {
+    const extension = moduleFileExtensions.find((extension) =>
+        fs.existsSync(resolveFn(`${filePath}.${extension}`))
+    )
+
+    if (extension) {
+        return resolveFn(`${filePath}.${extension}`)
+    }
+
+    return resolveFn(`${filePath}.js`)
+}
+
+export default {
+    appPath: resolveApp('.'),
+    projectPackageJson: resolveModule(resolveApp, 'package'),
+    appBuild: resolveApp('dist'),
+    formFieldProjectConfig: resolveModule(resolveApp, 'form-field.config'),
+    appRoot: resolveApp(''),
+    appSrc: resolveApp('src'),
+    appNodeModules: resolveApp('node_modules'),
+    devDist: resolveApp('node_modules/dev_dist'),
```

Comment:
dev_dist?

Reply:
We haven't used 'webpack-dev-server' which serves the bundle from computer's memory... We are building the project in dev mode (webpack is asked to place the dist inside /node_modules/dev_dist, because we don't want the user to see the dist generated), and spawning an http server with appropriate configuration on top of this directory - node_modules/dev_dist/